apps/dapp/src/components/Pages/Core/DappPages/Dashboard/Table/TxnHistoryTable.tsx

```diff
@@ -4,18 +4,65 @@ import { TxHistoryFilterType } from '.';
 import { DashboardType } from '../DashboardContent';
 import { format } from 'date-fns';
 import { TxnDataTable } from './TxnDataTable';
-import { PaginationControls } from './TxnPaginationControl';
+import { PaginationControl } from './PaginationControl';
 import { useTxHistory, useTxHistoryPaginationDefaults } from '../hooks/use-dashboardv2-txHistory';
 
 type Props = {
   dashboardType: DashboardType;
   filter: TxHistoryFilterType;
 };
 
+export type TxHistoryTableHeader = {
+  name: 'Type' | 'Date' | 'Debt Token' | 'Borrow' | 'Repay' | 'Tx Hash';
```

Comment:
I think it's better to have an `enum` with these table headers, to avoid mistakes and make it cleaner.